### PR TITLE
Change doc link for 15

### PR DESCRIPTION
**Pull request by kesselb**
Ref https://github.com/nextcloud/server/pull/12959

### CODE REVIEW

File: lib/private/legacy/defaults.php [hunk 1]
```diff
@@ -65,7 +65,7 @@ public function __construct() {
 		$this->defaultiTunesAppId = $config->getSystemValue('customclient_ios_appid', '1125420102');
 		$this->defaultAndroidClientUrl = $config->getSystemValue('customclient_android', 'https://play.google.com/store/apps/details?id=com.nextcloud.client');
 		$this->defaultDocBaseUrl = 'https://docs.nextcloud.com';
-		$this->defaultDocVersion = '14'; // used to generate doc links
+		$this->defaultDocVersion = '15'; // used to generate doc links
```
Comment (skjnldsv) [inline lib/private/legacy/defaults.php]: Please use 'stable' now :)
https://docs.nextcloud.com/server/stable/developer_manual/

**Review by skjnldsv — CHANGES_REQUESTED** [review]

File: lib/private/legacy/defaults.php (hunk 1 above)
Comment (rullzer) [reply lib/private/legacy/defaults.php]: @skjnldsv no on 15 we should use 15. Because once 16 is released we will forget to update this link.

Comment (skjnldsv) [reply lib/private/legacy/defaults.php]: @rullzer but then stable will be 16 :thinking:

Comment (danielkesselberg) [reply lib/private/legacy/defaults.php]: https://github.com/nextcloud/server/pull/12896 when you merge this one `defaultDocVersion` will always point to the right number. You cant forget it anymore :smile: Unfortunately #12896 didn't made it into `15` :disappointed:

Comment (rullzer) [reply lib/private/legacy/defaults.php]: @skjnldsv this PR is against stable15

**Review by rullzer — APPROVED** [review]

File: lib/private/legacy/defaults.php (hunk 1 above)
Comment (skjnldsv) [reply lib/private/legacy/defaults.php]: Ahaha, oups! :see_no_evil:

**Review by skjnldsv — APPROVED** [review]
hiding

**Pull request merged**